UnitTests/GitCommandsTests/Git/GitCommandHelpersTest.cs

```diff
@@ -59,6 +59,16 @@ public void CanGetRelativeNegativeDateString()
         }
 
         [TestMethod]
+        public void TestFetchArguments()
+        {
+            GitModule module = new GitModule(null);
+            var fetchCmd = module.FetchCmd("origin", "master", "local");
+            Assert.AreEqual("fetch --progress \"origin\" master --no-tags", fetchCmd);
```

Comment:
I believe you want this:
Assert.AreEqual("fetch --progress \"origin\" +refs/heads/master:local --no-tags", fetchCmd);
Do ypu want to fetch remote branch directly into local branch?

Reply:
No. The whole point of this change is _not_ to create a local branch on fetch! When you fetch master branch from origin, it updates the refs/remotes/origin/master. There's no need to create another ref for it.

I removed the localBranch argument which is not redundant.

Reply:
Sorry, but I don't understand the idea of your change. If you want ignore the localBranch argument, so what is the point of filling it out on the Fetch dialog?

Reply:
Changed this in the last revision. Now I removed the argument only for pull, but left it for fetch.